appdatabase/migrations/sql/1670249678_display_name_to_settings_sync_clock_table.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE settings_sync_clock ADD COLUMN display_name INTEGER NOT NULL DEFAULT 0;
```

Comment:
I'm not sure if this is necessary, but i have seen it done in other migrations
```suggestion
ALTER TABLE settings_sync_clock ADD COLUMN display_name INTEGER NOT NULL DEFAULT 0;
UPDATE settings_sync_clock SET display_name = 0;
```

Reply:
According to documentation it's not needed, you can check here: https://www.sqlite.org/lang_altertable.html
What should happen is:
```
sqlite> CREATE TABLE test (col1 INTEGER NOT NULL);
sqlite> INSERT INTO test VALUES(1);
sqlite> SELECT * FROM test;
1
sqlite> ALTER TABLE test ADD COLUMN new-col INTEGER DEFAULT 100 NOT NULL;
sqlite> SELECT * FROM test;
1|100
```

Reply:
Oh, i thought we won't use this method to solve the clock issue as discussed before😅 that's why my [pr](https://github.com/status-im/status-go/pull/2989) only in draft state